deploy/test/test_in_docker.sh

```diff
@@ -5,6 +5,9 @@ set -xeuo pipefail
 
 # Clean up when script completes and fails
 finish() {
+  dump_conjur_namespace_upon_error
```

Comment:
Thank you for putting this in!!!!

Reply:
I ended up removing it because I was getting "kubectl: command not found" errors... Not sure what that's about but I don't want it to hold this up.

Reply:
Ah, yeah, the 'kubectl' commands would need to be run via a test client container.